<launch>

  <node pkg="tf" type="static_transform_publisher" name="map_to_odom"
        args="0 0 0 0 0 0 /map /odom 100"/>

  <!-- Gmapping -->
  <node pkg="my_gmapping" type="my_gmapping"
        name="my_gmapping" output="screen">
    <!-- Frame IDs -->
    <param name="map_frame"  value="map"/>
    <param name="odom_frame" value="odom"/>
    <param name="base_frame" value="base_footprint"/>

    <!-- Map parameters -->
    <param name="map_x_max" value="1.0"/>
    <param name="map_x_min" value="-1.0"/>
    <param name="map_y_max" value="1.0"/>
    <param name="map_y_min" value="-1.0"/>
    <param name="map_resolution" value="0.1"/>

    <!-- Particle Filter parameters -->
    <param name="num_particles" value="10"/>
  </node>

</launch>